.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug Report
+description: Create a report for problems with ABACUS
+labels: Bug
+assignees: 
+  - dyzheng
+body: 
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >
```

Comment:
Using '>' in yaml may break the markdown grammar. Use '|' instead.
See https://yaml.org/spec/1.2.2/#scalars